src/pandapipes/toolbox.py

```diff
@@ -615,3 +615,40 @@ def get_internal_tables_pandas(net, convert_types=True):
                         tbl[col] = tbl[col].astype(np.bool_)
 
     return node_table, branch_table
+
+
+def print_pf_summary(net):
```

Comment:
We would need some kwargs here (e.g. for heating grids)

Reply:
We could also just choose between different modes here. But for a start, we can also just add a hint that this function returns some unuseful stuff for heating grids.